lib/src/remote_devtools_middleware.dart

```diff
@@ -101,14 +103,16 @@ class RemoteDevToolsMiddleware extends MiddlewareClass {
   }
 
   void _handleDispatch(dynamic action) {
+    if (this.store == null) {
```

Comment:
In my case it never reaches the code, when dispatching an action using the dev tools. The type is 'ACTION' and 'DISPATCH' is expected. 
I'm simply copying anther raw action, pasting it to the dispatch input and dispatching it. Should add something to the dispatched payload?

Reply:
I'll double check this today